src/rabbit_mgmt_stats_gc.erl

```diff
@@ -44,6 +45,8 @@
 
 -define(DROP_LENGTH, 1000).
 
+-define(PROCESS_ALIVENESS_TIMEOUT, 5000).
```

Comment:
We've tried 5 second timeouts for channel operations, they are too low for many. Please use 15s, in particular since this cannot be overridden, unlike channel operation timeouts.